tests/components/manual/test_alarm_control_panel.py

```diff
@@ -104,10 +105,10 @@ async def test_arm_home_with_pending(hass):
 
     await common.async_alarm_arm_home(hass, CODE, entity_id)
 
-    assert STATE_ALARM_PENDING == hass.states.get(entity_id).state
+    assert STATE_ALARM_ARMING == hass.states.get(entity_id).state
 
     state = hass.states.get(entity_id)
-    assert state.attributes["post_pending_state"] == STATE_ALARM_ARMED_HOME
+    assert state.attributes["next_state"] == STATE_ALARM_ARMED_HOME
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5411564)